src/debug/debug_gui.cpp

```diff
@@ -106,6 +107,14 @@ static list<string>::iterator logBuffPos = logBuff.end();
 
 extern int old_cursor_state;
 
+void savetologfile(const char *name) {
+    std::ofstream out(name);
```

Comment:
This is OK for now but in a future pull request please add a call to out.is_open() to confirm the ofstream instance was able to open the file without error.

https://www.cplusplus.com/reference/fstream/ofstream/is_open/